Tests/Schema.NET.Test/MixedTypesTest.cs

```diff
@@ -44,7 +44,7 @@ public class MixedTypesTest
         [Fact]
         public void DeserializeObject_WithCollectionOfMixedTypes_ReturnsObjectWithBothTypes()
         {
-            var book = JsonConvert.DeserializeObject<Book>(this.json, TestDefaults.DefaultJsonSerializerSettings);
+            var book = JsonConvert.DeserializeObject<Book>(this.json, TestDefaults.DefaultJsonSerializerSettings)!;
 
             Assert.True(book.Author.HasValue);

```

Comment:
Apparently `Assert.NotNull` supports nullable reference types, so if you assert that and use it's return value, you can avoid the exclamation here.

Reply:
Had a look but it doesn't seem to be the case - `Assert.NotNull` returns voids.

![image](https://user-images.githubusercontent.com/904226/111626499-02b99a00-883e-11eb-8ba8-55796b6a89c0.png)